rts.tests/supervise-base.sh

```diff
@@ -94,3 +94,38 @@ do
 done
 svc -u test.sv
 wait
+
```

Comment:
Please move this test into a new supervise-start.exp/sh file, and I'll pull the result.